spotseeker_server/management/commands/techloan/spotseeker.py

```diff
@@ -167,15 +169,25 @@ def _get_image_id(self, items: list, item_id) -> int:
             if item['id'] == item_id:
                 return item['images'][0]['id']
 
-    def _download_image(self, image_url, cte_type_id):
+    def _download_image(self, image_url, cte_type_id) -> str:
         try:
-            image, _ = urlretrieve(image_url)
-            return image
+            file_name, _ = urlretrieve(image_url)
+            return file_name
+        except HTTPError as e:
+            if e.code == 403:
+                r = requests.get(image_url, stream=True)
+                if r.status_code == 200:
+                    file_name = '/tmp/' + image_url.split('/')[-1]
```

Comment:
I assume the /tmp/ folder will always be there, and it's used by default for urlretrieve, but should this be looked into further?